xml/security_firewall.xml

```diff
@@ -993,7 +993,14 @@ nfs-rpc
         Monitor trace events in the <emphasis>nft</emphasis> tool and debugging and tracing in the rule set <emphasis>nftrace</emphasis>.
         </para>
       </listitem>
-        </itemizedlist>
+      <listitem>
+       <para>
+      The <command>nft</command> command line tool complies the rule set into a VM bytecode in netlink format.
+      During the rule set retrieval, the VM bytecode in netlink format is decompiled back to its original rule set representation.
+      nft acts as both a compiler and decompiler.
```

Comment:
`decompiling` is not really a common term in this context. I would rather says something like `is translated back into`.

Reply:
Could you please elaborate more , I didnt understand

Reply:
I would phrase it `The VM bytecode in netlink format is translated back into the rule set representation.`

Reply:
ok implemented , also removed the sentence, nft acts as both a compiler and decompiler , as its already explained